src/index.ts

```diff
@@ -265,6 +265,17 @@ export const envVarsConfig = {
 	DEVICE_TYPE_SPECIFIC_CONFIG_VAR_PROPERTIES,
 };
 
+// Needs to be `import *` so that the import path is preserved in the generated .d.ts
+import * as numericBigInteger from './translations/v6/numeric-big-integer-hack.js';
+/**
+ * This automatically augments the declarations of the consumer projects.
+ */
+export const augmentedTypes = {
+	'@balena/sbvr-types': {
+		'Numeric Big Integer': numericBigInteger,
```

Comment:
This allows consumers to have their v6 model typings work without having to manually import the numeric-big-integer-hack (or redeclare the augmented typed).
We can perhaps in the future can start adding more typings here which we atm have only in oB, like balena-device-config.ts or even common.ts